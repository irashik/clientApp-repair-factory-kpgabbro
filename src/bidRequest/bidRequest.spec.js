


import React from 'react';
import { shallow, render } from 'enzyme';

import BidRequest from './bidRequest';



describe('<BidRequest />', () => {

    test('snapShot correctly', () => {

        const component = shallow(<BidRequest />);
        expect(component).toMatchSnapshot();

    });
});


