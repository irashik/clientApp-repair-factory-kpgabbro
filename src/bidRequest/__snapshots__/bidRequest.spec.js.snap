// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BidRequest /> snapShot correctly 1`] = `
<Container
  className="m-0"
  fluid={true}
>
  <InputBidRequestForm
    handleAddedRecord={[Function]}
    onHide={[Function]}
    show={false}
  />
  <Row>
    <Col
      sm={4}
    >
      <label>
        Фильтр по статусу заявки
      </label>
      <StatusSelectField
        handleStatus={[Function]}
        statusBid=""
      />
    </Col>
    <Col>
      <label>
        Фильтр по категории
      </label>
      <CategorySelectField
        category=""
        handleCategory={[Function]}
      />
    </Col>
    <Col
      sm={4}
    >
      <label>
        Фильтр по приоритету
      </label>
      <PrioritySelectField
        handlePriority={[Function]}
        priority=""
      />
    </Col>
    <Col>
      <Button
        active={false}
        disabled={false}
        id="AddBidRequest"
        onClick={[Function]}
        variant="primary"
      >
        Добавить заявку
      </Button>
    </Col>
  </Row>
  <Row>
    <ReadListBidRequest
      addedRecord={Array []}
      category=""
      priority=""
      statusBid=""
    />
  </Row>
</Container>
`;
